mreg_cli/api/abstracts.py

```diff
@@ -384,7 +384,10 @@ def patch(self, fields: dict[str, Any], validate: bool = True) -> Self:
         :param validate: Whether to validate the patched object.
         :returns: The object refetched from the server.
         """
-        patch(self.endpoint().with_id(self.id_for_endpoint()), **fields)
+        if use_json:
+            patch(self.endpoint().with_id(self.id_for_endpoint()), fields, use_json=True)
+        else:
+            patch(self.endpoint().with_id(self.id_for_endpoint()), use_json=False, **fields)
```

Comment:
We need to pass in the `fields` argument as the `params` argument to `patch()` when using `use_json=True`, hence the need for differentiating them here...

Reply:
Always prior to this it seems? `patch` was called like this under the hood:

```py
requests.patch(path, data={"field": new_value"})
```

As opposed to this (used only when removing labels from what I can tell):

```py
requests.patch(path, json={"field": new_value"})
```

🤷

Reply:
... Well, that looks like something we probably should... fix. :D